app/views/advanced/_numismatics_advanced_search_form.html.erb

```diff
@@ -0,0 +1,62 @@
+<%= form_tag search_catalog_path, :class => 'advanced form-horizontal', :method => :get do  %>
```

Comment:
How is this partial used? Does it replace the existing numismatics search form component?

Reply:
I'm not sure I will look into this one.

Reply:
@sandbergja the partial was removed while I was away. Rebasing the branch fixed it.